src/vellum/utils/virtual_file_loader.py

```diff
@@ -0,0 +1,42 @@
+import importlib
+
+
+class VirtualFileLoader(importlib.abc.Loader):
```

Comment:
[nit] Could we leave docstrings in each class specifying that these are only meant for use by vellum? In fact, maybe we put it in the `ee` directory for now...

Reply:
Moved to ee/workflows/server. "vellum-use-only" doc string is overkill though, like most we already have tons of public stuff nobody would ever really use by themselves